airflow/providers/databricks/operators/databricks.py

```diff
@@ -303,7 +303,7 @@ def __init__(
         dbt_task: dict[str, str | list[str]] | None = None,
         new_cluster: dict[str, object] | None = None,
         existing_cluster_id: str | None = None,
-        libraries: list[dict[str, str]] | None = None,
+        libraries: list[dict[str, Any]] | None = None,
```

Comment:
```suggestion
        libraries: list[dict[str, dict[str, str]]] | None = None,
```
Shouldn't this be the typing, as per the ticket - https://github.com/apache/airflow/issues/31877 ?

Reply:
no, the value could be string sometimes, and dict sometime